Plugins/Mute/Commands/TempMuteCommand.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Text.RegularExpressions;
+using Data.Models.Client;
+using SharedLibraryCore;
+using SharedLibraryCore.Commands;
+using SharedLibraryCore.Configuration;
+using SharedLibraryCore.Interfaces;
+
+namespace Mute.Commands;
+
+public class TempMuteCommand : Command
+{
+    private const string TempBanRegex = @"([0-9]+\w+)\ (.+)";
+
+    public TempMuteCommand(CommandConfiguration config, ITranslationLookup translationLookup) : base(config,
+        translationLookup)
+    {
+        Name = "tempmute";
+        Description = translationLookup["PLUGINS_MUTE_COMMANDS_TEMPMUTE_DESC"];
+        Alias = "tmu";
+        Permission = EFClient.Permission.Moderator;
+        RequiresTarget = true;
+        SupportedGames = Plugin.SupportedGames;
+        Arguments = new[]
+        {
+            new CommandArgument
+            {
+                Name = translationLookup["COMMANDS_ARGS_PLAYER"],
+                Required = true
+            },
+            new CommandArgument
+            {
+                Name = translationLookup["COMMANDS_ARGS_DURATION"],
+                Required = true
+            },
+            new CommandArgument
+            {
+                Name = translationLookup["COMMANDS_ARGS_REASON"],
+                Required = true
+            }
+        };
+    }
+
+    public override async Task ExecuteAsync(GameEvent gameEvent)
+    {
+        var match = Regex.Match(gameEvent.Data, TempBanRegex);
+        if (match.Success)
```

Comment:
If they type `!tmu player 5 reason` they get no output. Let them know if the duration format is incorrect